.editorconfig

```diff
@@ -0,0 +1,23 @@
+# https://editorconfig.org/
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+[*.py]
+max_line_length = 100
+
+[Makefile]
+indent_style = tab
+
+[*.{json,y{a,}ml,sh}]
```

Comment:
for ``*.sh`` we prefer to use 4 spaces from my perspective. But I can live with 2 :-)

Reply:
I'm using two as well: https://github.com/TomasTomecek/dotfiles/blob/master/vim/.vimrc#L135